common-components/index.md

```diff
@@ -247,29 +249,33 @@ Below is an overview of some of the key benefits.
 - Testing and training for technical and user readiness
 
 **Cost benefits:**
-- Save money by consolidating support models, exiting from expensive exter-nal supplier contracts, and sharing central resources
+- Save money by consolidating support models, exiting from expensive external supplier contracts, and sharing central resources
 - Lowered cost of total ownership and economies of scale as multiple business areas contribute to maintenance costs
 
-##### Key documents
+__Documentation__
 Read the [Operating Model](https://educationgovuk.sharepoint.com/:w:/r/sites/lveesfa00073/_layouts/15/Doc.aspx?sourcedoc=%7b5E35BFD1-44C0-478B-81AA-368CAEED4CC3%7d&file=SharedCRM_Service_Operating_Model%20v0.1.docx&action=default&mobileredirect=true&CID=D33D9D75-C993-40C8-AAF5-449A95931FD9&wdLOR=c43C81DA6-1EF4-4B25-B4E7-F798DB011FE5) for more information about the solution design, security, availability, support and governance. If you need access to this document, please contact Matt Morgan.  
 
-##### Support
+__Support__
 The Enterprise CRM is supported internally by a central team within CIPS.  The team uses the ServiceNow toolset to manage the support of this service and follow DfE’s central service management processes.
 
-Contact the service desk to report an issue, the team are available from 08:00 – 22:00 hrs Monday to Friday, excluding English public holidays.  If the service desk is unable to resolve your query, the issue will be passed to a specialist technical team.  
+To report an issue with the service, log a ticket via the [Service Portal](https://dfe.service-now.com/serviceportal); the team are available from 08:00 – 22:00 hrs Monday to Friday, excluding English public holidays.  If the service desk is unable to resolve your query, the issue will be passed to a specialist technical team.  
+
+__Roadmap__
+DfE customers already using the Enterprise CRM or undergoing a transition to adopt the service include: 
+- National Apprenticeship Service (NAS – 267 users) 
```

Comment:
Numbers will quickly get out of date and probably not relevant - suggest remove?